UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentAncillaryFiles.cs

```diff
@@ -519,5 +525,27 @@ public async Task<bool> CreateCatalogFileForAio(string batchId, string exchangeS
             }
         }
 
+        /// <summary>
+        /// return cdType, BASE when base '0' folder and product with '.000' extension found in Aio Exchangeset
+        /// </summary>
+        /// <param name="salesCatalogueDataProductResponses"></param>
+        /// <param name="aioExchangeSetPath"></param>
+        /// <returns></returns>
+        private string GetCdType(List<SalesCatalogueDataProductResponse> salesCatalogueDataProductResponses, string aioExchangeSetPath)
+        {
+            string cdType = "UPDATE";
+            foreach (var response in salesCatalogueDataProductResponses)
+            {
+                string path = Path.Combine(aioExchangeSetPath, fileShareServiceConfig.Value.EncRoot, response.ProductName[..2], 
```

Comment:
Doesn't seem to be dependent on 'response' can you move this outside the foreach loop